sgx/seal.go

```diff
@@ -22,3 +22,23 @@ func SealToFile(data []byte, filePath string) error {
 
 	return nil
 }
+
+// Seal returns data sealed with unique ID in SGX-enabled environments
+// If SGX is disabled, it returns the data as is.
+func Seal(data []byte, enclaveEnabled bool) ([]byte, error) {
```

Comment:
I don't think it's good to have the `enclaveEnabled` param here. As I mentioned above, I think we don't need to make this project work in non-SGX environments.

Reply:
Let's have this as you did for now. I will check how we can improve our CI environment.